modules/visual_script/visual_script_builtin_funcs.cpp

```diff
@@ -64,6 +64,7 @@ const char* VisualScriptBuiltinFunc::func_name[VisualScriptBuiltinFunc::FUNC_MAX
 	"str2var",
 	"var2bytes",
 	"bytes2var",
+	"color_named",
```

Comment:
There was a comma on the previous last element, so I followed the style. I'm not changing it since "you don't like it", sorry (but, feel free to fix it everywhere when refactoring starts).